src/quo/components/list/item.cljs

```diff
@@ -105,14 +105,37 @@
                     :size                text-size}
          title]
         title)
-      (if (string? subtitle)
-        [text/text {:weight          :regular
-                    :color           :secondary
-                    :ellipsize-mode  :tail
-                    :number-of-lines subtitle-max-lines
-                    :size            text-size}
-         subtitle]
-        subtitle)]
+      (if (string? subtitle-secondary)
+        [rn/view {:flex-direction :row}
+         [text/text {:style {:max-width "56.5%"}
```

Comment:
could you please elaborate on the width value?

Reply:
Sure. Those are just to comply with the designs and assigning a maximum width for displaying long texts as the GfyCat Name and the chat key in the same line, with ellipsis rules and separated by a `•` char. The separator text width is fixed, so with the left and right texts max-width added equals to the 100% of the view's width. I can make these values settable if that makes more sense. Example below:

<img width="370" src="https://user-images.githubusercontent.com/18485527/113308228-587d5000-92dc-11eb-9d00-113985e1e602.png">